infrastructure/helm/quantumserverless/charts/gateway/templates/deployment.yaml

```diff
@@ -33,19 +33,19 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          command: [ "/usr/src/app/entrypoint.sh", "gunicorn", "main.wsgi:application", "--bind", "0.0.0.0:8000", "--workers=3" ]
+          args: [ "gunicorn", "main.wsgi:application", "--bind", "0.0.0.0:{{ .Values.service.port }}", "--workers=4" ]
```

Comment:
ah, that was a problem of not having migrations :)

Reply:
mostly, it seems that `command` overrides the `entrypoint` that you can have in your container (you always learn something new).